site/_apps/preact.txt

```diff
@@ -0,0 +1,14 @@
+---
+weight: 2
+title:  Preact
+logo: /assets/images/preact-logo.svg
+lighthouse: 99/100
+interactive-em: 2.4s
+interactive-faster-3g: 1.8s
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170423_RE_31X&run=1
+wpt-em-link: https://www.webpagetest.org/result/170420_73_54G/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170328_SX_2b62ae440379075b887f472c02a3e0a3/
+image: /assets/images/preacthn-mobile.png
+app-link: https://hn.kristoferbaxter.com/
```

Comment:
Thanks @kristoferbaxter, and you're right it only makes sense to add a link to their sources.

Added another button to view their GitHub repos.

<img width="1440" alt="screen shot 2017-04-24 at 6 27 20 pm" src="https://cloud.githubusercontent.com/assets/12476932/25361378/d435f9c4-291b-11e7-96ba-6f870992b1c3.png">
<img width="820" alt="screen shot 2017-04-24 at 6 27 44 pm" src="https://cloud.githubusercontent.com/assets/12476932/25361379/d4364c44-291b-11e7-96ec-15a47b895ddb.png">